python/remage/cli.py

```diff
@@ -181,6 +239,15 @@ def remage_run(
 
     args.append(f"--threads={threads}")
 
+    if merge_output_files:
+        args.append("--merge-output-files")
+
+    if reshape_output_files:
+        args.append("--reshape-output")
+
+    if time_window is not None:
+        args.append(f"--time-window-in-us {time_window}")
```

Comment:
```suggestion
        args.append(f"--time-window-in-us={time_window}")
```

otherwise it won't work - `args` is already an array of space-delimited strings.